dev-python/python-evdev/python-evdev-0.7.0-r1.ebuild

```diff
@@ -16,3 +16,9 @@ KEYWORDS="~amd64 ~arm ~x86"
 IUSE=""
 
 DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+
+python_compile() {
+	distutils-r1_python_compile build_ecodes \
+		--evdev-headers \
+		"${SYSROOT}"/usr/include/linux/input.h:"${SYSROOT}"/usr/include/linux/input-event-code.h
```

Comment:
SYSROOT is officially supported since EAPI 7. Could you EAPI-bump it then (with revbump)? Also check if this package has tests.

Reply:
Thanks for the note. My downstream has supported SYSROOT forever, so I didn't even notice the problem -- sorry!

This time I ran actual tests on a proper gentoo system, and I also noticed by typo too (s/code/codes/). All should be up to snuff now.